tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php

```diff
@@ -957,7 +980,7 @@ class A
                         const THREE = ONE + self::TWO; /* test */ # test
                         const B = 2;}
                 ',
-                ['elements' => ['const']],
+                ['elements' => ['const' => 'one']],
```

Comment:
`ClassAttributesSeparationFixer::SPACING_ONE` here or `one` in the changes lower - let's be consistent.